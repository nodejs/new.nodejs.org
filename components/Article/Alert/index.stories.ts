import Alert from './index';
import type { Meta as MetaObj, StoryObj } from '@storybook/react';

type Story = StoryObj<typeof Alert>;
type Meta = MetaObj<typeof Alert>;

export const Default: Story = {
  args: {
    children: 'This is an alert',
  },
};

export default { component: Alert } as Meta;
