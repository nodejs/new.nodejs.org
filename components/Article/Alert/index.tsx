import styles from './index.module.scss';
import type { FC, PropsWithChildren } from 'react';

const Alert: FC<PropsWithChildren> = ({ children }) => (
  <div className={styles.alert}>{children}</div>
);

export default Alert;
