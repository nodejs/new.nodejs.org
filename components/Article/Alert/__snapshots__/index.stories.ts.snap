// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article/Alert Default smoke-test 1`] = `
<div class="Alert_alert__qjYPQ">
  This is an alert
</div>
`;
